PyPortal/PyPortal_GCP_IOT_Planter/code.py

```diff
@@ -137,7 +145,14 @@ def handle_pump(command):
 
 
 # Initialize Google Cloud IoT Core interface
-google_iot = Cloud_Core(esp, secrets)
+settings = {
+    "cloud_region": getenv("cloud_region"),
+    "device_id": getenv("device_id"),
+    "private_key": getenv("private_key"),
```

Comment:
This one will need to be changed too. `private_key` in the current code is expected to be a tuple containing numbers that represent an RSA key. it's setup and putting into secrets.py is detailed here. https://learn.adafruit.com/pyportal-iot-plant-monitor-with-google-cloud-iot-core-and-circuitpython/code-setup#creating-a-rsa-key-pair-3038069

I think the best option for this is to create an `rsa_private_key.py` file that has a value with the key tuple in it. The code here can then import it like it was previously doing to secrets and use it.

It's kind of a shame to split the values between settings.toml and another file, but I do think it's best to keep everything that can use settings.toml in it, and just move the one that can't currently work to it's own file.

Reply:
We can add a placeholder `rsa_private_key.py` file to the repo that just contains a comment or string like "put RSA key values here" or similar.

Reply:
latest commit refactors this to it's own file and imports it from there.

Reply:
so the util prints:
```
private_key="(###, ###, ###, ###)"
```
so we can update to:
```
"private_key": eval(getenv("private_key")),
```
which will turn it into the tuple.

Does that work?

Reply:
I'd prefer not to use `eval()`